bloom-housing/bloom
# feat: feature flag migration script

This PR addresses [#(4460)](https://github.com/bloom-housing/bloom/issues/4460)


## Description

This script will create the 15 feature flags in use across Core, HBA and Doorway in each database. Each has a simplified name, a readable description and is set to false. Following release and running of this script, the API must be called to associate the needed jurisdictions.

Related links:
[Notion Documentation](https://www.notion.so/exygy/Environment-Variable-Creation-Best-Practices-1503433f03d080258bc4cdc9896f59b4?pvs=4)
[Existing Flags Sheet](https://docs.google.com/spreadsheets/d/105Srt0H_aYvb7DwONlwZl4AEN3aK8wU4kSFaIl2OcPc/edit?gid=0#gid=0)

## How Can This Be Tested/Reviewed?

- Run `yarn setup`
- Start up backend
- Sign in as an admin to the API
- Call `/scriptRunner/addFeatureFlags`
- Check console logs for success of all 15 flags
- Call GET `featureFlags/` to retrieve list of featureFlags

## Author Checklist:


## Review Process:

- Read and understand the issue
- Ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Either (1) explicitly ask a clarifying question, (2) request changes, or (3) approve the PR, even if there are very small remaining changes, if you don't need to re-review after the updates
